src/compliment/sources/namespaces_and_classes.clj

```diff
@@ -115,7 +117,8 @@
 
 (defn doc [ns-or-class-str curr-ns]
   (when (nscl-symbol? ns-or-class-str)
-    (let [ns-or-class-sym (symbol ns-or-class-str)]
+    (let [strip-literals #(-> % utils/split-by-leading-literals last)
```

Comment:
I think `second` instead of `last` would be a more common way to refer to a tuple part. `last` to me implies a variadic collection rather than a tuple.

Also, `(comp second utils/split-by-leading-literals)` would look more idiomatic.